heartbeat/heartbeat.py

```diff
@@ -0,0 +1,18 @@
+"""
+Code to keep the lambda function alive.
+"""
+import os
+
+from notifications_python_client.notifications import NotificationsAPIClient
+
+API_KEY = os.getenv("TF_VAR_heartbeat_api_key")
+# BASE_URL is a list of api urls - k8s and lambda.
+BASE_URL = os.getenv("TF_VAR_heartbeat_base_url")
```

Comment:
If this is an array, it would be nice to have a type declaration, i.e.
```suggestion
BASE_URL: [] = os.getenv("TF_VAR_heartbeat_base_url")
```

It would be nice to have it on all other constants too to be explicit defined too please. 😅 🙏